.eslintrc.js

```diff
@@ -1,13 +1,32 @@
-{
-  "root": true,
-  "env": {
-    "browser": true
+module.exports = {
+  globals: {
+    jest: "writeable",
+    describe: "readable",
+    it: "readable",
+    expect: "writeable",
+    afterEach: "writeable",
+    beforeEach: "writeable",
+    beforeAll: "readable",
+    afterAll: "readable"
```

Comment:
These globals are relevant only for test files, and can be achieved more easily with:
```js
{
  overrides: [
    {
      files: [testFilePattern],
      environment: {
        jest: true
      }
    }
  ]
}
```